source/addComment.js

```diff
@@ -1,3 +1,54 @@
+let addCommentBtn,
+    commentsCont
+
+const commentForm = `
```

Comment:
For "chunkier" html, maybe reach for adding to the HTML file and set its display to none or width to 0.